reference-adapter/pspcl-ifix-adapter/src/main/java/org/egov/entity/EventPostingDetail.java

```diff
@@ -0,0 +1,49 @@
+package org.egov.entity;
+
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+import java.util.Date;
+
+@Entity
+@Table(name = "pspcl_event_posting_detail")
```

Comment:
Referring relational db table in the model is not good.

Reply:
done.